### DISCUSSION TITLE
Is there a way to specify Swift versions from X.Y through to the latest?

### DISCUSSION THREAD

**Discussion by wadetregaskis** [swift-actions/setup-swift#559] [Q&A]
I'd like to be able to test with every major Swift release since a given anchor point (as defined by me in my workflow, based on the minimum Swift version required for my package).  e.g. 5.7+, meaning 5.7.1, 5.8, and - in future when it's release - 5.9 etc.

It seems I can only do one of two things, currently:

1. Specify no version, and just get whatever the latest happens to be.
2. Specify every desired version explicitly, and have to manually update those every time a new Swift version (including point releases) comes out.

**Comment by fwal** [comment]
Hi @wadetregaskis!

You can specify just the main version, in this case "5", to always get the latest version in the that version series (currently "5.8.1"). 🙂

Hope that helps!
